src/components/Form.tsx

```diff
@@ -1,62 +1,74 @@
 /* @jsx jsx */
 import { jsx } from '@emotion/core';
-import { Box } from '@stoplight/ui-kit';
 import * as React from 'react';
 
-import { useTheme } from '../theme';
-
 import { IFormtronControl } from '..';
 
 import { evaluate } from './evaluate';
+import { FieldSet } from './FieldSet';
+import { useDiagnostics } from './hooks';
+import { Messages } from './Messages';
+
+const substitute = (path: string, selection: string[]) => {
```

Comment:
Could be extracted to separate file or exported for better testing experience?
Also, it's unclear what this function does